src/server/server_family.cc

```diff
@@ -820,6 +822,12 @@ void SetSlowLogThreshold(util::ProactorPool& pool, int32_t val) {
   });
 }
 
+void SetMaxSquashedCmdNum(util::ProactorPool& pool, int32_t val) {
```

Comment:
Why did you decide to define and register it in server_family and not in main_service if we use it there?